src/http.rs

```diff
@@ -662,16 +679,17 @@ fn status_active(
             }).unwrap_or(-1),
         ).unwrap();
         if let Ok(duration) = metrics.duration {
-            writeln!(
+            write!(
                 res,
                 ", duration={:.3}s",
                 duration.as_secs() as f64
```

Comment:
maybe using `as_secs_f64()` can make this more concise

Reply:
Uh, didn’t even realize that existed.